BookingsAPI/Bookings.API/Controllers/HearingsController.cs

```diff
@@ -224,7 +225,10 @@ public async Task<IActionResult> UpdateHearingDetails(Guid hearingId, [FromBody]
 
             var command = new UpdateHearingCommand(hearingId, request.ScheduledDateTime,
                 request.ScheduledDuration, venue, request.HearingRoomName, request.OtherInformation,
-                request.UpdatedBy, cases, request.QuestionnaireNotRequired);
+                request.UpdatedBy, cases, request.QuestionnaireNotRequired)
+            {
+                StreamingFlag = request.StreamingFlag
```

Comment:
Why not passed into constructor ?

Reply:
@valentinabennett has convinced me 51% that its fine to not pass in to constructor as it false by default for the majority of cases